src/pfs/pfs.proto

```diff
@@ -472,6 +472,7 @@ message SQLDatabaseEgress {
         CSV = 1;
         JSON = 2;
         PARQUET = 3;
+        HEADER_CSV = 4;
```

Comment:
My reservation is that this could be a bit confusing to the user, there being two CSV formats: `csv` vs `header_csv`. What do you think about adding a `hasHeader` toggle instead?

Reply:
Updated to imply header row from presence of columns.